rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/SSHInfrastructureV2.java

```diff
@@ -109,6 +109,18 @@
     @Configurable(description = "Options for the java command launching the node on the remote hosts", sectionSelector = 3)
     protected String javaOptions;
 
+    @Configurable(description = "autoGenerated, useStartupScript or useNodeJarStartupScript. Please refer to the manual", sectionSelector = 1)
```

Comment:
we should decide which parameters are advanced, I would suggest all of the new parameters should be.

Reply:
Please refer to the manual => Please refer to the infrastructure documentation